### ISSUE TITLE
Both hands object response bug when using number keys

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#222]
Steps to reproduce:

1. Create screen
2. Show numbered tick marks
2. range to 0-30
4. focus both hands
5. press "0"
6. press "9"

-> object response is: `hands 9 tick marks apart from each other, left on tick mark 0, right on tick mark 9`

**Comment by zepumph** [comment]
This seems to fix it, but I don't know why yet. 


```
Index: js/common/view/RatioInteractionListener.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- js/common/view/RatioInteractionListener.js	(revision 011097f98195c1e5a989e8f850fffa5232a4b476)
+++ js/common/view/RatioInteractionListener.js	(date 1603838458258)
@@ -12,6 +12,7 @@
 import ratioAndProportion from '../../ratioAndProportion.js';
 import RAPConstants from '../RAPConstants.js';
 
+// TODO: rename to "BothHandsInteractionListener"
 class RatioInteractionListener {
 
   /**
@@ -113,7 +114,9 @@
           if ( event.key === i + '' ) {
             this.firstInteractionProperty.value = false;
             const newValue = 1 / this.tickMarkRangeProperty.value * i;
-            this.numeratorProperty.value = this.denominatorProperty.value = Utils.clamp( newValue, this.valueRange.min, this.valueRange.max );
+            const clamped = Utils.clamp( newValue, this.valueRange.min, this.valueRange.max );
+            this.numeratorProperty.value = clamped;
+            this.denominatorProperty.value = clamped;
 
             // If this brought to min or max, play the boundary sound
             if ( this.numeratorProperty.value === this.valueRange.min || this.numeratorProperty.value === this.valueRange.max ) {

```

**Comment by zepumph** [comment]
I found the bug to be that when the target ratio is 1/2, the unclamped fitness will be the same when at 0/N as it will for N/N, this is because of the fitness algorithm. Thus, listening just to fitness to rely on these changes was not enough. I updated the listener above, and documented it. It also struck me that it would likely be better to listen to the ratioTupleProperty instead, so I'm going to tackle that here too.
